app/javascript/controllers/facet_list_controller.js

```diff
@@ -0,0 +1,55 @@
+import { Controller } from "@hotwired/stimulus"
+
+(function() {
+  const BREAKPOINT = 1400; // xxl Bootstrap breakpoint
+  let isNarrow = window.innerWidth < BREAKPOINT;
+
+  window.addEventListener('resize', function() {
```

Comment:
Do we need to do this with javascript? What's javascript do for us that CSS can't?